up-l2/dispatchers/README.adoc

```diff
@@ -30,95 +30,88 @@ SPDX-License-Identifier: Apache-2.0
 
 .Dispatchers
 [width=100%",cols="30%,70%"]
 |===
 |Dispatcher uEs | Description
 
 |*uBus*
-|Message bus that dispatches CEs between uEs over a common transport. It provides multicast and forwarding functionality (works like a network switch)
+a|Local message bus responsible for dispatches messages between uEs within a uPlatform (uDevice namespace) using the same uTransport. Depending on the implementation, the uBus could either be implemented by the uTransport itself (e.g. zenoh based uPlatforms) or implemented in a standalone service (e.g. Android uPlatform)
 
 |*uStreamer*
-|Provides Device-2-Device CE routing either using the same or different transport protocols , i.e. when events need to move form one transport to the next it flows through the streamer (can be equated to an IP router)
+|Device-2-Device (uPlatform to uPlatform) message routing that over either the same or different uTransport
 
 |*Cloud Gateway*
 |A uE that sits at the edge of the cloud to connect non-cloud devices (ex. vehicles, phones, etc...) to the cloud
 
 |*Device Proxy Router (DPR)*
-|A uE that proxies CEs between devices that are not able to directly communicate with each other
+|A uE that proxies messages between devices that are not able to directly communicate with each other
 |===
 
 .Dispatchers
 image::dispatchers.drawio.svg[Dispatchers]
 
+
 == Requirements
 In this section we will elaborate on the requirements of the platform Dispatchers (uBus, uStreamer, etc...) and their role in message delivery. Dispatchers build upon the transport layer delivery requirement assumptions.
 
 NOTE: These communication layer requirements are still for point-2-point uE communication to and from a dispatcher
 
-* *MUST* support At-least-once delivery policy, this means that the dispatcher will make every attempt to dispatch the CE to the intended Receiver
-  ** *MUST* queue CEs not successfully acknowledged (transport level at-least-once delivery confirmation described above)
-  ** *MUST* attempt to retry transmission of the CE. Retry policy is specific to the dispatcher implementation
-  ** Dispatcher *MUST NOT* discard CEs unless either CE has expired (CE.ttl), or the egress queue is full. CEs that cannot be delivered are sent to a Dead Letter Office Topic
+* *MUST* support At-least-once delivery policy, this means that the dispatcher will make every attempt to dispatch the message to the intended receiver
+  ** *MUST* queue messages not successfully acknowledged (transport level at-least-once delivery confirmation described above)
+  ** *MUST* attempt to retry transmission of the message. Retry policy is specific to the dispatcher implementation
+  ** Dispatcher *MUST NOT* discard message unless either the message has expired (see https://github.com/eclipse-uprotocol/uprotocol-core-api/blob/main/src/main/proto/uattributes.proto[`UAttribute:ttl`] for more details), or the egress queue is full. Messages that cannot be delivered are sent to a Dead Letter Office Topic (DLT)
```

Comment:
Whenever we define a queue to buffer messages, we define a strategy for when the queue is full.
 Specifying the Dead Letter Queue is only one option and it can become full as well or blow up the system by hogging all the memory. 
There are many strategies that can be used and I think we should specify something here as well.
Priority Queue, Drop Oldest, Drop Newest ....